tests/db/test_db_storage.py

```diff
@@ -1,13 +1,17 @@
+import datetime
 import json
 from typing import Generator
 
 import pytest
 from prediction_market_agent_tooling.tools.utils import utcnow
+from sqlalchemy import Table
+from sqlmodel import SQLModel
 
 from prediction_market_agent.db.db_storage import DBStorage
+from prediction_market_agent.db.models import LongTermMemories, Prompt
 
 
-@pytest.fixture(scope="session")
+@pytest.fixture(scope="function")
```

Comment:
Changed scope to `function` to avoid handling setup and tear down, rather having a fresh DB for every test.